lua/telescope/builtin/git.lua

```diff
@@ -11,6 +16,10 @@ local conf = require('telescope.config').values
 
 local git = {}
 
+--- Lists files not ignored by git in your current directory
```

Comment:
Well it ignore not tracked files as well?

Reply:
I think it does ignore untracked files as well, I'll have to find out. I haven't used this picker before myself.